_pages/lfmp-2020-students.html

```diff
@@ -20,6 +20,17 @@ <h4><b>Nidhi Jain</b></h4>
       <h4><b>Dipanshu Verma</b></h4>
       <p><i>Wrapping proprietary printer drivers into a Printer Application</i></p>
     </div>
-   
+
+    <div class="col-lg-3 centered">
+      <img class="img img-circle" src="../assets/images/abhik.jpeg" alt="Abhik Chakraborty">
+      <h4><b>Dipanshu Verma</b></h4>
+      <p><i>IPP scan (or virtual MF device) server (Scanner Application)</i></p>
+    </div>
+
+    <div class="col-lg-3 centered">
+      <img class="img img-circle" src="../assets/images/rishabh.jpeg" alt="Rishab Arya">
+      <h4><b>Dipanshu Verma</b></h4>
```

Comment:
It needs Rishabh here.

Reply:
Thanks, Till, corrected the typo.

Reply:
I mean that you show the picture of Rishabh but the name displayed is Dipanshu.